projects/libs/core-library-typescript/src/virtual-machine/hard-coded-contracts/registry/kin-smart-contract.ts

```diff
@@ -0,0 +1,70 @@
+import BaseSmartContract from "../base-smart-contact";
+
+export default class KinAtnSmartContract extends BaseSmartContract {
+  static readonly ALLOCATED_ACCOUNT_NAME = "repository_source";
+  static readonly ONE_MILLION = 1000000;
```

Comment:
ONE_MILLION = 1000 x 1000 (more readable)

Reply:
BTW, in TypeScript you can do `1_000_000`.